bindings/Fortran/modules/adios2_adios_init_mod_mpi.F90

```diff
@@ -33,8 +33,7 @@ module adios2_adios_init_mod_mpi
 
 contains
 
-    ADIOS2_MODULE_PROCEDURE
-    subroutine adios2_init_mpi(adios, comm, adios2_debug_mode, ierr)
+    ADIOS2_MODULE_SUBROUTINE adios2_init_mpi(adios, comm, adios2_debug_mode, ierr)
```

Comment:
I guess I don't know whether this was needed (ie., whether there were problems with the continuation line?). But I'm pretty sure this is going to confuse editors that indent Fortran `subroutine` .. `end subroutine`.

I just tried, and yeah, at least emacs ends up being confused. It's a minor issue, but maybe worth considering.

Reply:
No issue with the continuation but the cray fortran compiler specifically has problems with the empty macro when submodules are disabled.

Reply:
The issue seems to be having an empty macro on a line by itself.  Does this work better in the editor?  It just works around the issue in a different way:
```fortran
#ifdef ADIOS2_HAVE_FORTRAN_SUBMODULES
# define ADIOS2_MODULE_PREFIX module
#else
# define ADIOS2_MODULE_PREFIX
#endif
...
contains
    ADIOS2_MODULE_PREFIX subroutine adios2_open_new_comm(engine, io, name, adios2_mode, comm, ierr)
...
    end subroutine
```

Reply:
`ADIOS2_MODULE_PREFIX subroutine ...` works for me -- I like it ;)